mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -211,11 +360,14 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	METHOD method_10234 (Lnet/minecraft/class_2378;)Lnet/minecraft/class_1792;
 		ARG 0 registry
 	METHOD method_10235 getIds ()Ljava/util/Set;
+		COMMENT {@return the set of all IDs registered in a registry}
 	METHOD method_10236 (Lnet/minecraft/class_2378;)Lnet/minecraft/class_2960;
 		ARG 0 registry
 	METHOD method_10237 (Lnet/minecraft/class_2378;)Lnet/minecraft/class_2248;
 		ARG 0 registry
 	METHOD method_10240 getRandom (Lnet/minecraft/class_5819;)Ljava/util/Optional;
+		COMMENT {@return a random registry entry from this registry, or an empty optional if the
```

Comment:
Should we mention that this should not be used if a registry has sensitive items, like the creative/operator items in the item registry, as this method polls without discrimination?

Reply:
Think this is fine. Operator only items have their own checks.